### DISCUSSION TITLE
2023-08-17

### DISCUSSION THREAD

**Discussion by MoonJeWoong** [woowacourse-teams/2023-haru-study#381] [daily_meeting]
### 개인 일정

- 노아: 5시 근로

### 오늘의 할 일

- 로그인 인가 기능 연동 마무리
- 발표 준비 및 체크리스트 (모디, 엽토)

#### 백엔드

- 프로필 요청시 memberId 제외하기
- 프로덕션 DB 마이그레이션
- 최종 서비스 배포

#### 프론트엔드

- 로그인 관련 남은 페이지 기능 구현

### 내일의 할 일

- 방학식
- 회식

### 언젠가 할 일

- 소켓, 폴링 양방향 통신 어떻게 할지 (4차 스프린트 내에 그냥 진행하고 발표만 안하는 방향으로 갈건지 로그인 이후 결정하기)
- 질문 PlaceHolder로 유의미한 응답 예시를 제공하는 기능 (목표설정 가이드라인)
- 브랜치 전략 결정하기(dev -> main)
- A라는 스터디를 참여후 B스터디 참여한 뒤 다시 A스터디에 참여할 시 닉네임을 찾지 못한 이슈(로그인이 된다면 해결될 문제이긴 함)

**Comment by MoonJeWoong** [comment]
### 체크인 
9점 - 어제 일찍 자고 일찍 일어나서 기분은 상쾌한데 오늘 마무리만 잘 하면 좋을 것 같네요~

**Comment by nlom0218** [comment]
### 체크인

10점 - 아침에 일찍 눈을 뜨니 좋군요

**Comment by jaehee329** [comment]
### 체크인
8점 - 빨리 방학좀!

**Comment by woosung1223** [comment]
### 체크인
8점 - 내일 방학이라서 기분이 싱숭생숭하군요

**Comment by yeopto** [comment]
### 체크인

7점 - 배고파요..

**Comment by woo-jk** [comment]
### 체크인
5점 - 어제 너무 늦게잠

**Comment by aak2075** [comment]
### 체크인

10점 - 벌써 레벨 3가 끝나가네요
